src/index.html

```diff
@@ -7,7 +7,56 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="container">
+      <div class="stars--0">
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+      </div>
+      <div class="stars--1">
+        <div class="stars__star_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+      </div>
+
+      <div class="stars--2">
+        <div class="stars__star_active"></div>
+        <div class="stars__star_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+      </div>
+
+      <div class="stars--3">
+        <div class="stars__star_active"></div>
+        <div class="stars__star_active"></div>
+        <div class="stars__star_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+      </div>
+
+      <div class="stars--4">
+        <div class="stars__star_active"></div>
+        <div class="stars__star_active"></div>
+        <div class="stars__star_active"></div>
+        <div class="stars__star_active"></div>
+        <div class="stars__star_not_active"></div>
+      </div>
+
+      <div class="stars--5">
+        <div class="stars__star_active"></div>
+        <div class="stars__star_active"></div>
+        <div class="stars__star_active"></div>
+        <div class="stars__star_active"></div>
+        <div class="stars__star_active"></div>
+      </div>
+
+    </div>
+
   </body>
```

Comment:
no needs to add empty lines before closing tag
```suggestion
      </div>
    </div>
  </body>
```